docs/scenario_editor/vehicle_models.md

```diff
@@ -0,0 +1,157 @@
+ 
+# New Vehicle Model 
+
+The monoDrive Scenario Editor provides users with the ability to import and export new vehicle models for more realistic simulation. By reducing the size of the polygon or triangle count, the user increases the processing speed for an asset in Unreal Engine. 
+
+## Reduction
+
+3D modeling software programs enable users to reduce the size of a model for a faster simulation process. 
+
+### Model Reduction from Maya
+
+1. When preparing the model for reduction, put the various parts of the model into combined groups, for example: Truck bed & Truck cab
+1. Set the origin points to the origin of the world (0,0,0)
+1. Export the model as FBX to UE4
+    - Check that parts are not too small or have single polygons, this could result in a less optimal result
+
+### UE4 Reduction
+
+ 1. Import all of the various parts as static meshes.
+ 
+ 1. Reduce a single mesh
+    1. Double click on a mesh to open.
+    1. On the right, under 'LOD settings' → 'Number of LODs'
+    1. Move the slider to the '2' position and click 'apply changes'
+        - In the top left, click 'LOD Auto' and change it to LOD 1
+        - the new model should be viewable and the reduction settings should be adjustable
+    1. On the Right, open the menu 'LOD 1' and click 'Reduction Settings' (If not found, use the search bar at the top)
+        - Adjust the 'Percent Triangles' slider to adjust the current LOD of the model
+        - Click apply to see the results
+
+Once all static meshes have been reduced, save the reduced asset. (picture of this as well)
+
+ 1. Drag a vehicle asset into the viewport
+ 1. Right click on the asset → 'Merge Actors'
+    - Open options and click on 'Use specific LOD level'
+    - Change the 'Specific LOD' option from 0 to 1
+    - Click 'Merge Actors'
+    - Select a location within UE4 to save the reduced asset
+
+### UE4 Reduction Export
+
+ 1. In the folder location where the exported asset(s) have been saved
+    - Right click on the asset
+    - Go to 'Asset Actions' → 'Export' 
+    - Select a location to save the reduced model
+    - These assets can be imported into any modeling program. For example: Autodesk Maya, Blender, Autodesk 3ds Max
+
+## Export
+
+Add introduction to the export process here. 
+
+[Blender Asset Export](https://continuebreak.com/articles/how-rig-vehicle-blender-28-ue4/)
+
+[Autodesk 3ds Max Asset Export](https://www.youtube.com/watch?v=tJL_AJbv_EU)
+
+
+### Export from Maya
+
+Before exporting:
+
+1. Combine all of the car parts into a single body except the tires.
+1. Name the body vehicle_root, and name the tires accordingly and as follows:
+    - FR_wheel
+    - FL_wheel
+    - BL_wheel
+    - BR_wheel
+
+    <div class='img_container'>
```

Comment:
Update picture from original documentation, move to be between numbers 5 & 6. Reference original documentation for positioning